src/Routes/__tests__/GSheetImport.test.ts

```diff
@@ -119,6 +121,7 @@ describe("GSheetImport", () => {
               groupType: GroupType.OtherCollections,
             },
           ],
+          featuredArtistExclusionIds: ["artist-id-for-pablo-picasso"],
```

Comment:
Kind of minor, but I think as a matter of communication to future us, this might be better as a BSON-looking id, assuming that's what we will ultimately use at the call site (compare to L106 above).